guide/03-the-gis/working-with-different-authentication-schemes.ipynb

```diff
@@ -401,7 +401,9 @@
    "cell_type": "markdown",
```

Comment:
<p style="user-select: auto;">I suggest changing this to "In our tests, we have been able to create a non-interactive login...... format."</p><p style="user-select: auto;"><br style="user-select: auto;"></p><p style="user-select: auto;">A non-interactive <s style="user-select: auto;">GIS object</s> login will look like below:</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>